ow_plexil/src/plans/plan-interface.h

```diff
@@ -26,6 +22,20 @@
 #define SHOULDER_PITCH 7
 #define SHOULDER_YAW   8
 
+#define NUM_JOINTS 9
+
+// Action goal statuses
+
+#define ACTION_PENDING 0
+#define ACTION_ACTIVE 1
+#define ACTION_PREEMPTED 2
+#define ACTION_SUCCEEDED 3
+#define ACTION_ABORTED 4
+#define ACTION_REJECTED 5
+#define ACTION_PREEMPTING 6
+#define ACTION_RECALLING 7
```

Comment:
I think Thomas mentioned that  ACTION_RECALLING, ACTION_RECALLED and ACTION_LOST are not currently supported in OW. It may be a good idea to mention this in a comment.

Reply:
Done, will be in next commit.